app/actions/DaemonActions.js

```diff
@@ -346,13 +346,18 @@ export const startWallet = (selectedWallet) => (dispatch, getState) => new Promi
     const lastPoliteiaAccessTime = walletCfg.get("politeia_last_access_time");
     const lastPoliteiaAccessBlock = walletCfg.get("politeia_last_access_block");
     const dismissBackupRedeemScript = walletCfg.get("dismiss_backup_msg_redeem_script");
+    const enablePrivacy = walletCfg.get("enableprivacy");
+    const mixingAccount = walletCfg.get("mixingaccount");
```

Comment:
Probably best to follow dcrwallet's convention here and call this the "mixedAccount"